app/src/main/java/com/kickstarter/libs/utils/extensions/ProjectExt.kt

```diff
@@ -319,5 +325,6 @@ fun Project.reduceToPreLaunchProject(): Project {
         .currency(this.currency())
         .currencySymbol(this.currencySymbol())
         .currencyTrailingCode(this.currencyTrailingCode())
+        .urls(Urls.builder().web(web).build())
```

Comment:
This method returns a Project with only the properties needed. We need to include the webProjectUrl property so that the ref tag is correctly passed along by RefTagUtils: https://github.com/kickstarter/android-oss/blob/26b84b02b9d0f63079aba8c08f2b4dee9313c4fd/app/src/main/java/com/kickstarter/libs/utils/RefTagUtils.java#L89